test/SendRecv_OutOfPlace.cpp

```diff
@@ -0,0 +1,90 @@
+/*************************************************************************
+ * Copyright (c) 2022 Advanced Micro Devices, Inc. All rights reserved.
+ *
+ * See LICENSE.txt for license information
+ ************************************************************************/
+#include "TestBed.hpp"
+
+namespace RcclUnitTesting
+{
+  TEST(SendRecv, OutOfPlace)
+  {
+    TestBed testBed;
+
+    // Configuration
+    std::vector<ncclFunc_t>     const  funcType        = {ncclCollSend, ncclCollRecv}; // akollias
+    std::vector<ncclDataType_t> const& dataTypes       = {ncclInt32};
+    std::vector<ncclRedOp_t>    const& redOps          = {ncclSum}; //Not important for send receive tests
+    std::vector<int>            const  numElements     = {1024}; // akollias, one test for now
+    bool                        const  inPlace         = false;
+    bool                        const  useManagedMem   = false;
+    int                         const  numCollPerGroup = numElements.size();
+
+    // CollFuncPtr prepFunc = DefaultPrepData_SendRecv; // akollias
+    bool isCorrect = true;
+    // for (int totalRanks = testBed.ev.minGpus; totalRanks <= testBed.ev.maxGpus && isCorrect; ++totalRanks) // big iterator for all ranks
+    int totalRanks = testBed.ev.maxGpus; // akollias this to change on maxGpus
+      int const numProcesses = 1;
+      int const isMultiProcess = 0;
+      testBed.InitComms(TestBed::GetDeviceIdsList(numProcesses, totalRanks), numCollPerGroup);
+
+      // for (int redOpIdx = 0; redOpIdx < redOps.size() && isCorrect; ++redOpIdx)
+      for (int dataIdx = 0; dataIdx < dataTypes.size() && isCorrect; ++dataIdx)
+      {
+        if (testBed.ev.showNames) // Show test names
+          INFO("%s process %2d-ranks SendRec v%d out Of place (%s)\n",

```

Comment:
Drop the number of ranks, and show the (SRC -> DST) pair instead (Rank 0 -> Rank 3)